testnet-contracts/contracts/Oracle.sol

```diff
@@ -0,0 +1,297 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+import "../../node_modules/openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "./Recoverer.sol";
+
+contract Oracle is Recoverer {
+
+    using SafeMath for uint256;
+
+    /*
+    * @dev: Public variable declarations
+    */
+    address public provider;
```

Comment:
Provider is confusing as its a defined term in the web3 space with web3Provider. Should we use operator rather?